### PR TITLE
[13.0][MIG] partner_contact_configuration

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

File: partner_contact_configuration/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+some users don't have sales rights and you want them to access partners
+configuration (title, tags, ...)
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
```
Comment (nikul-serpentcs) [inline partner_contact_configuration/README.rst:19]: @Laurent-Corron 13.0

File: partner_contact_configuration/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+{
+    "name": "Partner Contact Configuration",
+    "summary": """
+        Adds menu configuration access through the 'contacts' module main menu
+        """,
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://acsone.eu",
```
Comment (nikul-serpentcs) [inline partner_contact_configuration/__manifest__.py:12]: https://github.com/OCA/partner-contact

**Review by nikul-serpentcs — COMMENTED** [review]
Netpicking

**Review by lmignon — CHANGES_REQUESTED** [review]
@Laurent-Corron @pedrobaeza IMO this addon is no more required into odoo 13.0 .....
Titles and Tags are already available into a configuration menu under Contacts. This addon introduces duplicate entries for the same functionalities...
Default menu
image


Menu added by this addon
image

**Comment by pedrobaeza** [comment]
I only did the administrative work of classifying the PR, but I agree with that facts that is not needed.

**Comment by Laurent-Corron** [comment]
Sorry, this was a mistake by my part, I didn't notice that it was already available in "Configuration", closing this MR.

**Pull request closed without merging**